autogen.sh

```diff
@@ -1,5 +1,19 @@
-#! /bin/sh
+#! /bin/bash
 
 test -d ./config || mkdir ./config
 
+FILES=./man/*.1
+printf "dist_man1_MANS = " > ./man/Makefile.am
+for f in $FILES
+do
+  printf "%s " $(basename $f) >> ./man/Makefile.am
+done
+
+FILES=./man/*.3
+printf "\ndist_man3_MANS = " >> ./man/Makefile.am
+for f in $FILES
+do
+  printf "%s " $(basename $f) >> ./man/Makefile.am
+done
+
```

Comment:
Append a newline to ```man/Makefile.am``` here, otherwise some tools may complain that the line is unterminated.